internal/cli/atlas_alerts_acknowledge.go

```diff
@@ -42,35 +43,42 @@ func (opts *atlasAlertsAcknowledgeOpts) initStore() error {
 }
 
 func (opts *atlasAlertsAcknowledgeOpts) Run() error {
-	body := opts.newAcknowledgeRequest()
+	body, err := opts.newAcknowledgeRequest()
+	if err != nil {
+		return err
+	}
 	result, err := opts.store.AcknowledgeAlert(opts.ProjectID(), opts.alertID, body)
-
 	if err != nil {
 		return err
 	}
 
 	return json.PrettyPrint(result)
 }
 
-func (opts *atlasAlertsAcknowledgeOpts) newAcknowledgeRequest() *atlas.AcknowledgeRequest {
+func (opts *atlasAlertsAcknowledgeOpts) newAcknowledgeRequest() (*atlas.AcknowledgeRequest, error) {
+	if opts.forever && opts.until != "" {
+		return nil, errors.New("--forever and --until are exclusive")
```

Comment:
let's do this with flags.Forever and flags.Until
```suggestion
		return nil, fmt.Errorf("--%s and --5s are exclusive", flags...)
```

Reply:
Also this kind of validation is better to do it in the `PreRunE` section of the command so cobra can show suggestions but this is not necessary for this PR